web/auth/guardian_serializer.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CodeCorps.GuardianSerializer do
+  @behaviour Guardian.Serializer
+
+  alias CodeCorps.Repo
+  alias CodeCorps.User
+
+  def for_token(user = %User{}), do: { :ok, "User:#{user.id}" }
+  def for_token(_), do: { :error, "Unknown resource type" }
+
+  def from_token("User:" <> id), do: { :ok, Repo.get(User, id) }
```

Comment:
@green-arrow wouldn't we need the user's information still though for something like `canary`. Sure, we could encode role and access information into the token, but it's entirely possible the token does not have the same level of information about the user as our database. For most authenticated requests, won't we want that level of information about the user?